src/target/linux.rs

```diff
@@ -417,7 +417,11 @@ mod tests {
 
         let threads: Vec<_> = threads
             .iter()
-            .map(|t| (t.name().unwrap().unwrap().clone(), t.thread_id()))
+            .map(|t| {
+                let name = t.name().unwrap().unwrap_or("".to_string());
```

Comment:
```suggestion
                let name = t.name().unwrap().unwrap_or_else(String::new);
```

Prevents an allocation in the happy case.